.github/workflows/dotnet-format.yml

```
@@ -23,8 +23,5 @@ jobs:
     - name: Setup dotnet
       uses: actions/setup-dotnet@v3
 
-    - name: Install format tool
-      run: dotnet tool install -g dotnet-format
-
     - name: dotnet format
-      run: dotnet-format --folder --check
+      run: dotnet format opentelemetry-dotnet-contrib.sln --no-restore --verify-no-changes
```

For sure you are using wrong sln file. Probably you should use.

```suggestion
      run: dotnet format OpenTelemetry.sln --no-restore --verify-no-changes
```

What is more, if you do not touch any *.cs file, the step will be ignored in CI.
Please find some tiny improvement in any *.cs file and include it into PR. It can be even in comment.

---

Oh ! Ok let me make those changes .

---

@SaumyaBhushan I just tried running `dotnet format OpenTelemetry.sln --no-restore --verify-no-changes` locally on the command line from the repo root and it is spitting out warnings. If you do that exercise locally it will give you some .cs files to modify (to fix the warnings) 😄 We'll have to get to 0-warnings before we can make this change.